src/pages/surveys/[id]/questions/[order].page.tsx

```diff
@@ -0,0 +1,79 @@
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useContext, useEffect, useState } from 'react';
+
+import Button from 'components/Button';
+import PageLoader from 'components/PageLoader';
+import SurveyQuestion from 'components/SurveyQuestion';
+import useSession from 'hooks/useSession';
+import { SurveyQuestion as SurveyQuestionInterface } from 'services/surveyInterfaces';
+import { StoreContext } from 'store';
+
+import styles from './[order].module.css';
+
+const Question: NextPage = () => {
+  useSession();
+  const router = useRouter();
+  const questionOrder = parseInt(router.query.order as string);
+  const surveyId = router.query.id as string;
+
+  const {
+    store: { currentSurvey },
+  } = useContext(StoreContext);
+
+  const [currentQuestion, setCurrentQuestion] =
+    useState<SurveyQuestionInterface>({} as SurveyQuestionInterface);
+  const [isLoading, setIsLoading] = useState(true);
+  const [answers, setAnswers] = useState(null);
+
+  useEffect(() => {
+    const question = currentSurvey?.questions.find(
+      ({ displayOrder }) => displayOrder === questionOrder
+    );
+
+    if (question !== undefined) {
+      setCurrentQuestion(question);
+      setIsLoading(false);
+    } else {
+      router.push(`/surveys/${surveyId}`);
+    }
+  }, [questionOrder]);
+
+  const totalQuestion = currentSurvey?.questions.length || 0;
+
+  return (
+    <PageLoader isLoading={isLoading}>
+      <>
+        <div className="h-full w-full max-w-3xl mx-auto flex flex-col justify-center">
+          <div className="mb-4 text-gray-400">{`${questionOrder}/${totalQuestion}`}</div>
```

Comment:
Should we hide this as well when unsupported type?

Reply:
@hoangmirs, I think we can keep the question order because it is easier to see which question order is the unsupported one.